apps/elewa-website/src/app/app.module.ts

```diff
@@ -1,22 +1,16 @@
-import { NgModule } from '@angular/core';
+import {  NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { HttpClientModule } from '@angular/common/http';
 
 import { AppComponent } from './app.component';
 import { TranslocoRootModule } from './transloco-root.module';
-import { AppHeaderModule } from '@elewa-website/elements/layout/header';
 
 import { AppRoutingModule } from './app.routing';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [
-    BrowserModule,
-    AppRoutingModule,
-    HttpClientModule,
-    TranslocoRootModule,
-    AppHeaderModule,
-  ],
+  imports: [BrowserModule, AppRoutingModule, HttpClientModule, TranslocoRootModule,AppHeaderModule],
   providers: [],
```

Comment:
This reverts really important changes in the codebase. Kindly pull from main and resolve the conflicts again.